spec/features/collections/collection_results_spec.rb

```diff
@@ -103,7 +103,7 @@
   it "doesn't' show version_id for collections that don't have one" do
     fill_in "keywords", with: 'C1214605943-SCIOPS'
     wait_for_xhr
-    expect(page).to have_no_content("vNot provided")
+    expect(page).to have_no_content("Not provided")
```

Comment:
This change was a random typo I found and decided to clean up as part of the ticket.